docs/product/data-management-settings/filtering/index.mdx

```diff
@@ -16,14 +16,22 @@ Inbound filters include:
 
 - Common browser extension errors
 - Events coming from localhost
-- Known legacy browsers errors
+- From legacy browsers
 - Known web crawlers
 - By their error message
 - From specific release versions of your code
 - From certain IP addresses
 - React hydration errors
 - Transactions coming from healthcheck and ping requests
```

Comment:
I would reorder these to follow the same order as the UI:


- Common browser extension errors
- Transactions coming from health checks and ping requests
- Events coming from legacy browsers
- Events coming from localhost
- Errors from known web crawlers
- React hydration errors
- ChunkLoadErrors
- Events from certain IP addresses
- Events with certain error messages
- Events from specific release versions of your code